pushsync/protocol.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package pushsync
+
+import (
+	"crypto/rand"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+const (
+	pssChunkTopic   = "PUSHSYNC_CHUNKS"   // pss topic for chunks
+	pssReceiptTopic = "PUSHSYNC_RECEIPTS" // pss topic for statement of custody receipts
+)
+
+// PubSub is a Postal Service interface needed to send/receive chunks and receipts for push syncing
+type PubSub interface {
+	Register(topic string, prox bool, handler func(msg []byte, p *p2p.Peer) error) func()
```

Comment:
in the spirit of calling the interface `PubSub` I would expect it to have 2 methods:
1. `Publish`
2. `Subscribe`
Please rename either the methods or the interface (although `RegSen` is probably less sexy)
Also, maybe `to` should be an `Address` rather than a byte slice?

Reply:
non of this was addressed (nor commented on). is this such a seemingly unrelated question to the scope of this pr?